assets/js/index.js

```diff
@@ -0,0 +1,901 @@
+		if (modal.dataset.modalTargetString) {
+			var modalTargetClass = modal.dataset.modalTargetString;
+			modalToggle = document.querySelector('*[data-toggle-target="' + modalTargetClass + '"]');
+		}
+
+		// If a modal toggle exists, trigger it so all of the toggle options are included
+		if (modalToggle) {
+			modalToggle.click();
+
+			// If one doesn't exist, just hide the modal
+		} else {
+			modal.classList.remove('active');
+		}
+	},
+
+}; // twentytwenty.coverModals
+
+/*	-----------------------------------------------------------------------------------------------
+	Dynamic Screen Height
+--------------------------------------------------------------------------------------------------- */
+
+twentytwenty.dynamicScreenHeight = {
+
+	init: function () {
+		this.setScreenHeight();
+
+		setTimeout(function () {
+			this.setScreenHeight();
+		}.bind(this), 500);
+
+		window.addEventListener('resize', function () {
+			this.setScreenHeight();
+		}.bind(this));
+	},
+
+	setScreenHeight: function () {
+		var screenHeight = document.querySelector('.screen-height');
+		if (screenHeight) {
+			screenHeight.style.minHeight = window.innerHeight;
+		}
+	},
+
+}; // twentytwenty.dynamicScreenHeight
+
+/*	-----------------------------------------------------------------------------------------------
+	Focus Management
+--------------------------------------------------------------------------------------------------- */
+
+twentytwenty.focusManagement = {
+
+	init: function () {
+		// If the visitor tabs out of the main menu, return focus to the navigation toggle
+		// Also, if the visitor tabs into a hidden element, move the focus to the element after the hidden element
+		this.focusLoop();
+	},
+
+	focusLoop: function () {
+		document.addEventListener('focusin', function (event) {
+			var element = event.target;
+			var menuModal = document.querySelector('.menu-modal');
+			var headerToggles = document.querySelector('.header-toggles');
+			var searchModal = document.querySelector('.search-modal');
+			if (menuModal.classList.contains('.active')) {
+				if (!menuModal.contains(element) && !headerToggles.contains(element)) {
+					document.querySelector('.nav-toggle').focus();
+				}
+			} else if (!searchModal.classList.contains('.active')) {
+				if (!searchModal.contains(element)) {
+					searchModal.querySelector('.search-field').focus();
+				}
+			}
+		});
+	},
+
+}; // twentytwenty.focusManagement
+
+/*	-----------------------------------------------------------------------------------------------
+	Intrinsic Ratio Embeds
+--------------------------------------------------------------------------------------------------- */
+
+twentytwenty.intrinsicRatioVideos = {
+
+	init: function () {
+		this.makeFit();
+
+		window.addEventListener('fit-videos', function () {
+			this.makeFit();
+		});
+
+		window.addEventListener('resize', function () {
+			this.makeFit();
+		});
```

Comment:
There's a scope issue here, it should be something like:
```
window.addEventListener('resize', function () {
    this.makeFit();
}.bind(this));
```
or just use `this.makeFit` as the callback

Reply:
@carolinan also suggested that `add_theme_support( 'responsive-embeds' );` would allow us to remove this block altogether.

Reply:
`responsive-embeds` is available only in WP 5.*. Shouldn't it be backwards compatible?

Reply:
The theme review rules only allow us to be backwards compatible three major versions, which will be 5.0 when 5.3 is out.

Reply:
I guess so :) I thought this being a core theme should go even further...

Reply:
It's going to be set as a Featured theme and it will be in top 3 popular themes. Some of the user might find it that way and use it on older versions of WP

Reply:
`responsive-embeds` was used in Twenty Nineteen as well, so there's some precedent for it. But I see your point.